index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Ghormeh Sabzi - Persian Herb Stew</title>
+</head>
+<body>
+
+  <header id="header" class="pxy-1">
+    <h1><span class="secondary-text">Ghormeh Sabzi</span></h1>
+    <nav class="navbar">
+      <ul class="nav-list">
+        <li><a href="#header">Home</a></li>
+        <li><a href="#ingredients">Ingredients</a></li>
+        <li><a href="#recipe">Recipe</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main id="main">
+    <!-- Showcase Section -->
+    <section class="showcase .pxy-1">
+      <h2>Ghormeh Sabzi - Persian Herb Stew</h2>
+      <p>You will get familiar with this delicious dish</p>
+      <a href="#ingredients">Let's start</a>
+    </section>
+
+    <!-- Ingredients Section -->
+    <section  id="ingredients" class="ingredients pxy-1 mxy-1">
+        <div class="des-context col">
+          <h3> <span class="primary-text"><strong>Ingredients</strong> </span></h3>
+          <p> <span class="secondary-text ">Serves 4-6</span></p>
+          <ul>
+            <li>2 pounds meat (lamb or beef) washed and cubed</li>
+            <li>1 cup red kidney beans or pinto beans, soaked overnight</li>
+            <li>1 large yellow onion, finely chopped</li>
+            <li>4 bunches flat-leafed parsley</li>
+            <li>1 bunch cilantro</li>
+            <li>2 large leeks or 4 scallions (green stems only)</li>
+            <li>1 small bunch fresh fenugreek, or 1 tablespoon dried fenugreek, too much fenugreek will cause ghormeh sabzi to taste bitter</li>
+            <li>3-4 dried lemons, or 3 tablespoons freshly squeezed lemon juice</li>
+            <li>1 teaspoon turmeric</li>
+            <li>Salt and pepper to taste</li>
+            <li>Vegetable oil</li>
+            <li>Water</li>
+          </ul>
+        </div>
+        <div class="des-pic col">
+        </div>
+    </section>
+
+    <!-- Recipe Section -->
+    <section id="recipe" class="recipe pxy-1 mxy-1">
+        <h3 class="text-center"> <span class="primary-text"><strong>Recipe</strong></span></h3>
+      <ol>
+        <li>Clean and wash the herbs, dry thoroughly and chop finely. </li>
+        <li>Heat 1/3 cup of vegetable oil in a large skillet over medium heat and saute the chopped fresh herbs for about 15 minutes, stir frequently. Set aside.</li>
```

Comment:
we can use html special characters instead of `1/3` see https://www.w3schools.com/charsets/ref_html_entities_4.asp

Reply:
Thank you for your guidance. I corrected my code